docs/en/introduction/adopters.md

```diff
@@ -10,6 +10,7 @@ toc_title: Adopters
 
 | Company | Industry | Usecase | Cluster Size | (Un)Compressed Data Size<abbr title="of single replica"><sup>\*</sup></abbr> | Reference |
 |---------|----------|---------|--------------|------------------------------------------------------------------------------|-----------|
+| <a href="https://www.sangfor.com.cn/" class="favicon">sangfor</a> | Subscription Analytics | Main product | — | — | [Tweet, November 2021](https://www.sangfor.com.cn/) |
```

Comment:
So where exactly did they mention ClickHouse usage in their Twitter in November 2021? Did not find anything related.

Reply:
And in any case, `sangfor` should also be placed in alphabetical order exactly below S7 Airlines.